### PR TITLE
fix: min stake ratio

### PR DESCRIPTION
**Pull request by Longarithm**

### CODE REVIEW

File: core/primitives/src/epoch_manager.rs [hunk 1]
```diff
@@ -297,6 +299,12 @@ impl AllEpochConfig {
         }
     }
 
+    fn config_fix_min_stake_ratio(config: &mut EpochConfig, protocol_version: u32) {
+        if checked_feature!("stable", FixMinStakeRatio, protocol_version) {
+            config.validator_selection_config.minimum_stake_ratio = Rational32::new(1, 62_500);
+        }
+    }
+
```
Comment (jancionear) [inline core/primitives/src/epoch_manager.rs:307]: I don't have the knowledge to verify that it's the correct way to do it, but I assume that you know what you're doing. It's weird that we modify both the json file and the code?

**Review by jancionear — APPROVED** [review]
Generally lgtm, although I'm not an expert on validator assignments.

File: core/primitives/src/epoch_manager.rs (hunk 1 above)
Comment (Longarithm) [reply core/primitives/src/epoch_manager.rs:307]: We are in the middle of changing source data from genesis config to small json configs, so for now both modifications are required.

File: core/primitives-core/src/version.rs [hunk 2]
```diff
@@ -147,6 +147,7 @@ pub enum ProtocolFeature {
     EthImplicitAccounts,
     /// Enables yield execution which is introduced in <https://github.com/near/NEPs/pull/519>
     YieldExecution,
+    FixMinStakeRatio,
```
Comment (wacban) [inline core/primitives-core/src/version.rs:150]: nit: Can you add a short comment and maybe a link if there is a good full description of the problem somewhere?

**Review by wacban — APPROVED** [review]
LGTM

File: core/primitives-core/src/version.rs (hunk 2 above)
Comment (Longarithm) [reply core/primitives-core/src/version.rs:153]: Yeah, done.

**Pull request merged**